### PR TITLE
Cleanup Range structure

### PR DESCRIPTION
**Pull request by guillaumekln**
* Remove unused _p_start attribute
* Remove leading underscore for public attributes
* Remove constructor

### CODE REVIEW

File: include/fuzzy/ngram_matches.hh [hunk 1]
```diff
@@ -9,17 +9,17 @@
 
 namespace fuzzy
 {
-  class Range {
-  public:
-    Range(size_t p_start,
-          size_t suffix_first, size_t suffix_last,
-          size_t match_length):_p_start(p_start), _suffix_first(suffix_first),
-                               _suffix_last(suffix_last), _match_length(match_length)
+  struct Range {
```
Comment (ClementChouteau) [inline include/fuzzy/ngram_matches.hh:12]: Maybe we should remove this constructor ?

Comment (guillaumekln) [reply include/fuzzy/ngram_matches.hh:12]: It seems to be useful to initialize all attributes in one call. What do you think?

Comment (ClementChouteau) [reply include/fuzzy/ngram_matches.hh:12]: It seems redundant with something like this `Range({suffix_first, suffix_last, match_length})`

Comment (guillaumekln) [reply include/fuzzy/ngram_matches.hh:12]: Right, this works too. I'll do that.

**Pull request merged**